src/components/shared/form_fields/RadioField.vue

```diff
@@ -143,7 +143,10 @@ const onFieldChange = ( newValue: string | number | boolean | null ): void => {
 				padding: 0 map.get(units.$spacing, 'xx-small') 0;
 				margin-bottom: map.get(units.$spacing, 'small');
 
-				&:last-child {
+				/* We remove the margin from all items on the last row */
+				&:last-child,
+				&:nth-child( odd ):last-child,
+				&:nth-child( odd ):nth-last-of-type( 2 ) {
```

Comment:
Whoa, this took some serious thinking to decipher this, but now I (almost) get it:

- `:last-child` is for the last element (right column)
- `:nth-child( odd ):nth-last-of-type( 2 )` is for the 2nd-last element (left column). We need two pseudo-selectors for cases where we have an odd number of elements (i.e. one element in the last row, on the left column), to prevent the `nth-last-of-type(2)` pseudoselector to affect the right-column element in the row above
- `:nth-child( odd ):last-child` is also for cases where we have an odd number of elements. I just wonder if we can't leave this out because the `:last-child` pseudoselector covers both cases. We could probably save a few nanoseconds and bytes by removing this.

Did I explain everything right?

Reply:
If there's an odd number of items we need to remove the margin from only the last item. If it's an even number then we need to remove it from the last 2 items. It broke my brain a little too, let me take another look